### PR TITLE
Sponsorship blog post

### PR DESCRIPTION
**Pull request by agagata**

### CODE REVIEW

**Comment by bmispelon** [comment]
Looks good to me :+1:

File: tinyblog/articles/2016-06-07-we-need-your-help.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# We need your help✋
+
+> Published on 7th Jun 2016 by Agata Grdal
+
+We are seeking companies who wish to support Django Under the Hood financially.
```
Comment (mjtamlyn) [inline tinyblog/articles/2016-06-07-we-need-your-help.md]: We usually style this as `Django: Under the Hood`

File: tinyblog/articles/2016-06-07-we-need-your-help.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+---
+
+As organizers, we are focused on making this event the best place to share knowledge and build a community around Django.
+
+We want to gather Django core team members and Django developers to teach us all about Django. We want to welcome the most diverse group of people possible. And we want to make the event as affordable and accessible as possible for everyone.
```
Comment (mjtamlyn) [inline tinyblog/articles/2016-06-07-we-need-your-help.md]: Drop the `And`.

File: tinyblog/articles/2016-06-07-we-need-your-help.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+We know those are ambitious goals. In order to achieve them, we need your support.
+
+You can find our sponsorship brochure here: <a href="http://tinyurl.com/duth2016-sponsorship">http://tinyurl.com/duth2016-sponsorship</a>, but we’re also open to new ideas.
```
Comment (mjtamlyn) [inline tinyblog/articles/2016-06-07-we-need-your-help.md]: Maybe `any of your ideas`?

**Comment by olasitarska** [comment]
Merged! Thanks Agata :)

**Pull request merged**